Legion/VaultOfTheWardens/Tormentorum.lua

```diff
@@ -129,4 +132,28 @@ do
 			self:Message(args.spellId, "Personal", "Alert", CL.underyou:format(args.spellName))
 		end
 	end
+end
+
+do
+	local prev = 0
+	local list = mod:NewTargetList()
+	function mod:FleshtoStoneApplied(args)
+		local amount = args.amount or 1
+		if amount < 7 then

```

Comment:
No point in registering `SPELL_AURA_APPLIED` if we return on 1 anyway. Looks like registering `SPELL_AURA_APPLIED_DOSE` and just using `args.amount` is what you want to do here.